package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "node api/index.js", 
+    "dev": "nodemon index.js",
+    "test": "npm run test:js && npm run lint",
+    "test:js": "tape 'test/**/*.test.js' | tap-spec",
```

Comment:
this command says "look for all the files in the `test/` folder that end with `.test.js` and run those tests"
The `| tap-spec` bit on the end means "and pipe the results to a nice output printer which gives us nice :heavy_check_mark: marks